.github/workflows/library.yml

```diff
@@ -366,6 +366,7 @@ jobs:
 
           git remote set-url origin "https://github.com/${REPO}.git"
           git fetch origin $BRANCH
+git checkout $BRANCH
```

Comment:
Haha... Oops... Fixing things on the phone... The lack of indent here will break most of the CI :/